### PR TITLE
chore: cleanup and dependency updates

### PR DESCRIPTION
**Pull request by rebelchris**
Cleanup some false test cases, update dependencies

### CODE REVIEW

File: 1683868176744-PostMentionTracking.ts [hunk 1]
```diff
@@ -1,17 +0,0 @@
-import { MigrationInterface, QueryRunner } from 'typeorm';
```
Comment (idoshamun) [inline 1683868176744-PostMentionTracking.ts:1]: Why did you delete that migration?

File: __tests__/cron/exportToTinybird.ts [hunk 2]
```diff
@@ -67,7 +72,7 @@ describe('PostsRepository', () => {
       post_private: 0,
       post_type: 'article',
       source_id: 'a',
-      source_type: 'Source',
+      source_type: 'machine',
```
Comment (sshanzel) [inline __tests__/cron/exportToTinybird.ts:75]: Good find with all these.

File: 1683868176744-PostMentionTracking.ts (hunk 1 above)
Comment (rebelchris) [reply 1683868176744-PostMentionTracking.ts:1]: Was situated in the root of the project, not migration folder.
AFAIK they don't get triggered anyway. (and looked to be outdated)

Comment (sshanzel) [reply 1683868176744-PostMentionTracking.ts:1]: Oh, in that case, I think we might have to move it to the correct migrations folder if the replica identity is not accurate with `PostMention`.

File: __tests__/comments.ts [hunk 3]
```diff
@@ -169,9 +174,10 @@ const saveSquadFixture = async (sourceId: string) => {
       image: 'sample2/image',
     },
   ]);
-  await con
-    .getRepository(Source)
-    .save(createSource(sourceId, 'A', 'http://a.com', SourceType.Squad, true));
+  await con.getRepository(Source).update({ id: sourceId }, { private: true });
+  // await con
+  //   .getRepository(Source)
+  //   .save(createSource(sourceId, 'A', 'http://a.com', SourceType.Squad, true));
```
Comment (sshanzel) [inline __tests__/comments.ts:180]: I think we can now remove this comment.

**Review by sshanzel — COMMENTED** [review]
Always nice to keep things updated. Just had a few comments but nothing blocking :shipit:

**Review by sshanzel — APPROVED** [review]
Again, thanks for doing the dependency updates! 🚀
